test/models/piece_test.rb

```diff
@@ -1,7 +1,18 @@
 require 'test_helper'
 
 class PieceTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @user1 = FactoryGirl.create(:user)
+    @user2 = FactoryGirl.create(:user)
+    @piece1 = Piece.create(type: "Pawn", row_position: 4, col_position: 2, user_id: @user1.id, captured: false)
+    @piece2 = Piece.create(type: "Pawn", row_position: 5, col_position: 1, user_id: @user2.id, captured: false)
+  end
+
+  test "capture logic" do
```

Comment:
I think this test's name should be more descriptive.  For example, it could be `"pawn is captured with move_to!"`.